scripts/release_scripts/generate_release_updates.py

```diff
@@ -134,6 +134,7 @@ def prompt_user_to_send_announcement_email():
         'Please make sure to check that the mail ids of new authors '
         'and contributors are correct.\n' % (
             RELEASE_MAIL_MESSAGE_FILEPATH, new_contributors_mail_ids))
+    common.ask_user_to_confirm('Add announcements label to the email sent.\n')
```

Comment:
We only want the `announcement` label in the oppia@ email, right? I would suggest opening the group in the browser (i.e, do line 139 before this) and ask the user to add the label.

Reply:
Done.